templates/create-daml-app/package.json

```diff
@@ -0,0 +1,28 @@
+{
+    "scripts": {
+        "test": "yarn workspace create-daml-app run test"
```

Comment:
You seem to hardcode the name `create-daml-app` here while making it a template variable. As a first step, I would just not make it a variable. That should be fine for the GTG (but not for starting a new project) and saves you the trouble of making it configurable everywhere.

Reply:
Ok, so in that case if I say `daml new my-app create-daml-app` it will create the directory `my-app` but still use `create-daml-app` as the project name throughout, right?

Reply:
Yes, not great for use as a template but a useful starting point and easier to implement.

Reply:
Also once you have tests working it'll be a lot easier to make a PR to parameterize the name and check you've not forgotten some instances.